### ISSUE TITLE
After some scroll to up and down images not showing

### ISSUE THREAD

**Issue by orazz** [Baseflow/flutter_cached_network_image#229]
After some scroll gridview up and down images not showing

```
new CachedNetworkImage(
                  imageUrl: widget.product.image,
                  placeholder: (context, url) => Center(
                    child: Container(height: 30, width: 30,
                                  child: new CircularProgressIndicator(
                        valueColor: new AlwaysStoppedAnimation<Color>(Colors.black12),
                      ),
                    ),
                  ),
                  errorWidget: (context, url, error) {
                    print("image - $error");
                    return Center(child: Text(error));
                  },
          ),
```
<img src="https://user-images.githubusercontent.com/8214953/63740040-ef845980-c8a8-11e9-895d-5373bc7e4fc3.png" height="300"> <img src="https://user-images.githubusercontent.com/8214953/63740042-ef845980-c8a8-11e9-8d2f-618f9ea2a9e0.png" height="300">

lib version: cached_network_image: ^1.1.1

**Comment by renefloor** [comment]
I guess this is fixed in 2.3.0-beta, but there is an issue still when using an error widget. Would be nice if you can validate without using an error widget.

**Comment by orazz** [comment]
Thanks, It works!
